Formula/zellij.rb

```diff
@@ -0,0 +1,27 @@
+class Zellij < Formula
+  desc "Pluggable terminal workspace, with terminal multiplexer as the base feature"
+  homepage "https://zellij.dev"
+  url "https://github.com/zellij-org/zellij/archive/refs/tags/v0.8.0.tar.gz"
+  sha256 "850d1a68219debdb4e05f7f58e70b31f440425247caaa1b29b9be1b6c7869207"
+  license "MIT"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
```

Comment:
Like mentioned above
```suggestion
```

Reply:
Oh! thanks I thought I commited this in https://github.com/Homebrew/homebrew-core/commit/91ae4d640cfd47cfa5813afa6d50500ed9b0c5ee